### ISSUE TITLE
request for pushing/nudging

### ISSUE THREAD

**Issue by itsdeadfrosty** [Traben-0/solid_mobs#39]
like vanilla, mods should be able to be bumped into to push them out of the way. this also allows players to positions pets and such properly

**Comment by bfreezie** [comment]
Yeah, I'll add my voice to this one. Turning the platform mode off in the config offers better compatibility with Create contraptions and the ships from Small ships, but that comes with the downside of having every mob turn into an unmovable object. It would be nice if mobs could be nudged in either mode.

**Comment by UndoZan** [comment]
So im guessing with platform mode turned mobs are moveable? What is platform mode? Sorry if it is described ingame, im creating a private modpack and im not launching the game after every mod i install

**Comment by Traben-0** [comment]
platform mode is a setting in solid mobs to only have collisions with the top of mobs

as for puushing this mod does provide an option for that shift and right clicking a mob will nudge it
